packages/module/src/components/CatalogTile/CatalogTileBadge.tsx

```diff
@@ -28,7 +28,7 @@ export const CatalogTileBadge: React.FunctionComponent<CatalogTileBadgeProps> =
         <Tooltip id={id} content={title}>
           <span className={classes} {...props}>
             {children}
-            <span className="pf-v5-u-screen-reader">{title}</span>
+            <span className="pf-v6-u-screen-reader">{title}</span>
```

Comment:
Assuming these components import PF's base styles, any use of `pf-v6-u-screen-reader` can be updated to use `pf-v6-screen-reader`.